nuggetsapp/models.py

```diff
@@ -59,5 +66,21 @@ def add_new_nugget_for_user(cls, nugget, user, is_owner=False):
 
     @classmethod
     def get_nuggets_by_user(cls, user):
-        return [x.nugget for x in cls.objects.filter(user = user)]
+        return [x.nugget for x in cls.objects.filter(user = user).filter(is_deleted = False)]
+        
+    @classmethod
+    def get_todays_review_nuggets_by_user(cls, user):
+        today = datetime.date.today(); 
+        one_day_back = today - timedelta(days=1); 
+        three_days_back = today - timedelta(days = 3); 
+        one_week_back = today - timedelta(days = 7); 
+        two_weeks_back = today - timedelta(days = 14); 
+        one_month_back = today - timedelta(days = 30); 
+        three_months_back = today - timedelta(days = 3 * 30)
+        six_months_back = today - timedelta(days = 6 * 30)
+        one_year_back = today - timedelta(days = 12 * 30)
+        two_years_back = today - timedelta(days = 24 * 30)
+ 
+        return [x.nugget for x in  cls.objects.filter(Q(user = user) & Q(is_deleted = False) & (Q(created_at__date__gte = one_day_back) | Q(created_at__date__in = [three_days_back, one_week_back, two_weeks_back, one_month_back, three_months_back, six_months_back, one_year_back, two_years_back])))]
```

Comment:
@nathanwchan the __date apparently strips out the date from date time. 
`created_at__date__gte`